pkgs/applications/editors/neovim/default.nix

```diff
@@ -16,7 +21,10 @@ let
     ++ optionals doCheck [
         nvim-client luv coxpcall busted luafilesystem penlight inspect
       ]
+    ++ optional devMode luacheck
```

Comment:
So here, I'd suggest:

```suggestion
    ++ optional doCheck luacheck
```

Reply:
luacheck is just for linting, it's not used during the functionaltest (it's deceptive :) )